cmd/podman/pull.go

```diff
@@ -163,7 +168,7 @@ func pullCmd(c *cliconfig.PullValues) error {
 		for _, name := range names {
 			newImage, err := runtime.New(getContext(), name, c.String("signature-policy"), authfile, writer, &dockerRegistryOptions, image.SigningOptions{}, true, nil)
 			if err != nil {
-				println(errors.Wrapf(err, "error pulling image %q", name))
+				logrus.Errorf("error pulling image %q", name)
```

Comment:
We still get proper verbose errors with log-level=debug enabled, right?

Reply:
yes, we get the same errors as before.  The println(...) statement was only causing to print something like "(0x1840f00,0xc00041bba0)"